utils/envvars/parser.go

```diff
@@ -135,6 +142,33 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value.
+// if it's valid, otherwise returns defaultSAASHost.
+func GetSAASHost(env string) string {
+	if v, b := os.LookupEnv(envSAASHost); b {
+		if u := govalidator.IsURL(v); u {
+			logrus.Warnf("SAAS host changed to %s.", v)
+
+			return v
+		}
+		logrus.Warnf("environment variable %s has invalid format %s.", env, v)
+	}
+
+	if v, b := os.LookupEnv(env); b {
+		if u := govalidator.IsURL(v); u {
+			logrus.Warnf("environment variable %s WILL BE REMOVED SOON, please use %s instead", env, envSAASHost)
```

Comment:
Log messages should start with a capital letter and end with a dot.

Reply:
done thanks